src/templates/digitalMedia/digitalMedia.scss

```diff
@@ -0,0 +1,147 @@
+/* Custom styling for digital specimen page */
+
+.digitalMedia_body {
+    height: Calc(100% - 147px);
+}
+
+.digitalMedia_mediaBody {
+    height: 70%;
+}
+
+.digitalMedia_titleBlock {
+    height: 40px;
+    line-height: 40px;
+    font-size: 20px;
+}
+
+.digitalMedia_mediaBodyImages {
+    height: Calc(100% - 40px) !important;
+}
+
+.digitalMedia_mediaBodyTab {
+    height: 40px;
```

Comment:
Still a lot of fixed heights based on px. How will that scale if on smaller/bigger screens?

Reply:
This is an probable issue, also in connection with the whole mobile capacity of UCAS, which is minimal in the current situation

Reply:
Ok something to keep in mind but which we cannot solve in this PR.